config/manifests/bases/submariner.clusterserviceversion.yaml

```diff
@@ -17,7 +17,304 @@ metadata:
+    owned:
+    - description: Broker is the Schema for the brokers API.
+      displayName: Submariner Broker
+      kind: Broker
+      name: brokers.submariner.io
+      resources:
+      - kind: Deployment
+        name: submariner-operator
+        version: v1
+      specDescriptors:
+      - description: List of the components to be installed - any of [service-discovery,
+          connectivity].
+        displayName: Components
+        path: components
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+      - description: List of domains to use for multi-cluster service discovery.
+        displayName: Default Custom Domains
+        path: defaultCustomDomains
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+      - description: Default cluster size for GlobalCIDR allocated to each cluster
+          (amount of global IPs).
+        displayName: Default Globalnet Cluster Size
+        path: defaultGlobalnetClusterSize
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:number
+        - urn:alm:descriptor:com.tectonic.ui:fieldDependency:globalnetEnabled:true
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+      - description: GlobalCIDR supernet range for allocating GlobalCIDRs to each
+          cluster.
+        displayName: Globalnet CIDR Range
+        path: globalnetCIDRRange
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+        - urn:alm:descriptor:com.tectonic.ui:fieldDependency:globalnetEnabled:true
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+      - description: Enable support for Overlapping CIDRs in connecting clusters.
+        displayName: Enable Globalnet
+        path: globalnetEnabled
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:booleanSwitch
+      version: v1alpha1
+    - description: ServiceDiscovery is the Schema for the servicediscoveries API.
+      displayName: Service Discovery
+      kind: ServiceDiscovery
+      name: servicediscoveries.submariner.io
+      specDescriptors:
+      - description: Name of the custom CoreDNS configmap.
+        displayName: CoreDNS Custom Config Name
+        path: coreDNSCustomConfig.configMapName
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+      - description: Namespace of the custom CoreDNS configmap.
+        displayName: CoreDNS Custom Config Namespace
+        path: coreDNSCustomConfig.namespace
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+      version: v1alpha1
+    - description: Submariner is the Schema for the submariners API.
+      displayName: Submariner
+      kind: Submariner
+      name: submariners.submariner.io
+      resources:
+      - kind: Deployment
+        name: submariner-operator
+        version: v1
+      specDescriptors:
+      - description: Type of broker (must be "k8s").
```

Comment:
@skitt didn't we drop this one?

Reply:
Not yet, I’ve started removing uses of the field, we need to remove all of those before we can drop it here.